packages/expect/src/types.ts

```diff
@@ -83,6 +113,8 @@ export type Expect = {
   stringMatching(expected: string | RegExp): AsymmetricMatcher;
   [id: string]: AsymmetricMatcher;
   not: {[id: string]: AsymmetricMatcher};
+  matchers: MatchersObject;
+  print: PrintObject;
```

Comment:
yeah, not a huge fan as it doesn't really belong on the `expect` function... We might need to defer this to the next major when we can migrate to ESM style exports (still transpiled to CJS, tho).

One alternative is to create a new package (`@jest/expect-utils` or some such) which exports them for public consumption and use within `expect` itself. Not a huge fan of that either TBH 😅 So this might need to wait for Jest 28

Reply:
Fair enough!

After it kicks around the back of your brain a bit more, let me know if you see another way forward. In the meantime, if necessary, consumers can always patch. 🩹